From 835ead88a3e474f3c0bb09de21cb9333a4f1613c Mon Sep 17 00:00:00 2001
From: Viresh Kumar <viresh.kumar@st.com>
Date: Thu, 29 Mar 2012 11:03:27 +0530
Subject: [PATCH 154/611] Documentation: pinctrl: add missing spi0_0 grp in
 example

Missed one group from the documentation when proofreading.

Signed-off-by: Viresh Kumar <viresh.kumar@st.com>
Signed-off-by: Linus Walleij <linus.walleij@linaro.org>
(cherry picked from commit eb181c3533e8e5b3eaa96d3e8ed1cf5d9a408076)

Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 Documentation/pinctrl.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/pinctrl.txt b/Documentation/pinctrl.txt
index f5add79..2c6897c 100644
--- a/Documentation/pinctrl.txt
+++ b/Documentation/pinctrl.txt
@@ -635,7 +635,7 @@ struct foo_pmx_func {
 	const unsigned num_groups;
 };
 
-static const char * const spi0_groups[] = { "spi0_1_grp" };
+static const char * const spi0_groups[] = { "spi0_0_grp", "spi0_1_grp" };
 static const char * const i2c0_groups[] = { "i2c0_grp" };
 static const char * const mmc0_groups[] = { "mmc0_1_grp", "mmc0_2_grp",
 					"mmc0_3_grp" };
-- 
1.8.2

